shell/platform/windows/angle_surface_manager.cc

```diff
@@ -190,6 +199,20 @@ bool AngleSurfaceManager::CreateSurface(WindowsRenderTarget* render_target) {
   return true;
 }
 
+void AngleSurfaceManager::ResizeSurface(EGLint width, EGLint height) {
+  EGLint existing_width, existing_height;
+  GetSurfaceDimensions(&existing_width, &existing_height);
+  if (width != existing_width || height != existing_height) {
+    // This resize approach could be further optimized if Angle exposed a public
+    // entrypoint for SwapChain11::reset or SwapChain11::resize.
+    DestroySurface();
+    if (!CreateSurface(current_render_target_, width, height)) {
```

Comment:
Does this destroy/create pair discard the previous drawing, or is the previous output preserved by the render target? I'm wondering if we need to blit the old content into the new surface synchronously to prevent flashing.

If we don't need to do that, maybe a comment here explaining why would be helpful.

Reply:
Fixed in next commit